Content.Server/Chemistry/ReagentEffects/PlantMetabolism/RobustHarvest.cs

```diff
@@ -16,12 +16,18 @@ public override void Effect(ReagentEffectArgs args)
                                     plantHolderComp.Seed.Immutable)
                 return;
 
+            const int potencyLimit = 50;
```

Comment:
This should probably be a field on the effect so we can have weaker or stronger reagents with this effect. Same for the hardcoded potency increase